app/src/cdogsService/routes.js

```diff
@@ -26,11 +14,11 @@ routes.get('/', (req, res) => {
   });
 });
 
-routes.get('/health', keycloak.protect(protector), (req, res, next) => {
+routes.get('/health', authenticate, (req, res, next) => {
   healthCheck(req, res, next);
 });
 
-routes.post('/template/render', keycloak.protect(protector), (req, res, next) => {
```

Comment:
I think this keycloak-protect() function we were using before was only verifying the jwt was from the realm in `server.keycloak` properties.

But with the new authenticate() function we are instead verifying the jwt came from the realm and has `audience` matching attributes of `frontend.keyclaok`.
But I think the new pattern is correct.